### PR TITLE
Initial readme.md file taken and modified from Rust rfcs repo.

### PR DESCRIPTION
**Pull request by chick**
Included empty .gitignore for good measure

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
+# Chisel RFCs - [Active RFC List](https://rfcbot.rs/)
```
Comment (jackkoenig) [inline README.md]: I would do Chisel/FIRRTL or Chisel-Lang, we definitely want to make this obvious that it's not just Chisel

I would also delete the link to https://rfcbot.rs/, that's the real list of Rust RFCs

File: README.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+  - [RFC Postponement]
+  - [Help this is all too informal!]
+  - [License]
+
+
+## When you need to follow this process
+[When you need to follow this process]: #when-you-need-to-follow-this-process
+
+You need to follow this process if you intend to make "substantial" changes to
+Chisel, Cargo, Crates.io, or the RFC process itself. What constitutes a
```
Comment (jackkoenig) [inline README.md]: Cargo and Crates.io are Rust things

File: README.md [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+"substantial" change is evolving based on community norms and varies depending
+on what part of the ecosystem you are proposing to change, but may include the
+following.
+
+  - Any semantic or syntactic change to the language that is not a bugfix.
+  - Removing language features, including those that are feature-gated.
+  - Changes to the interface between the compiler and libraries, including lang
+    items and intrinsics.
+  - Additions to `std`.
```
Comment (jackkoenig) [inline README.md]: Change `std` to `util` I guess, or `chisel3.util`

File: README.md [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+    if it is made prematurely.
+    - For RFCs with lengthy discussion, the motion to FCP is usually preceded by
+      a *summary comment* trying to lay out the current state of the discussion
+      and major tradeoffs/points of disagreement.
+    - Before actually entering FCP, *all* members of the team must sign off;
+    this is often the point at which many team members first review the RFC
+    in full depth.
+  - The FCP lasts ten calendar days, so that it is open for at least 5 business
+    days. It is also advertised widely,
+    e.g. in [The Chisel Blog](https://chisel.eecs.berkeley.edu/blog/). This way all
```
Comment (jackkoenig) [inline README.md]: + chisel-users mailing list probably

File: README.md [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+    is reached.
+  - In most cases, the FCP period is quiet, and the RFC is either merged or
+    closed. However, sometimes substantial new arguments or ideas are raised,
+    the FCP is canceled, and the RFC goes back into development mode.
+
+## The RFC life-cycle
+[The RFC life-cycle]: #the-rfc-life-cycle
+
+Once an RFC becomes "active" then authors may implement it and submit the
+feature as a pull request to one of the Chisel repos. Being "active" is not a rubber
```
Comment (jackkoenig) [inline README.md]: I think most mentions of "Chisel" should be "Chisel-Lang"

File: README.md [hunk 6]
```diff
@@ -0,0 +1,256 @@
...
+
+
+[developer discussion forum]: http://internals.chisel-lang.org/
+[RFC issue tracker]: https://github.com/freechipsproject/rfcs/issues
+[RFC repository]: http://github.com/freechipsproject/rfcs
+
+## License
+[License]: #license
+
+This repository is currently in the process of being licensed under
```
Comment (jackkoenig) [inline README.md]: Not in the process of, lets make this repo Apache 2.0 from the start

**Comment by chick** [comment]
@jackkoenig try  number 2

File: README.md [hunk 7]
```diff
@@ -0,0 +1,270 @@
...
+
+### Help this is all too informal!
+[Help this is all too informal!]: #help-this-is-all-too-informal
+
+The process is intended to be as lightweight as reasonable for the present
+circumstances. As usual, we are trying to let the process be driven by
+consensus and community norms, not impose more structure than necessary.
+
+
+[developer discussion forum]: http://internals.chisel-lang.org/
```
Comment (azidar) [inline README.md]: This link doesn't work

**Review by azidar — APPROVED** [review]

File: README.md (hunk 7 above)
Comment (chick) [reply README.md]: re-pointed at mail group

File: README.md (hunk 6 above)
Comment (chick) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (chick) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (chick) [reply README.md]: Added chisel users google group

File: README.md (hunk 2 above)
Comment (chick) [reply README.md]: Removed

File: README.md (hunk 1 above)
Comment (chick) [reply README.md]: Went with chisel-lang
Deleted rfcbot.rs

File: README.md (hunk 3 above)
Comment (chick) [reply README.md]: done

**Pull request merged**